File: PRAK204-2510817110006-Muhammad Fadhil Lesmana.py
#Program menghitung: a - b * i / j - x + y

def hitung_ekspresi(a, b, i, j, x, y):
    hasil = a - b * i / j - x + y
    return round(hasil, 3)

#Input dari user
data = input("Masukkan 6 angka (a b i j x y): ").split()
a, b, i, j, x, y = map(float, data)

hasil = hitung_ekspresi(a, b, i, j, x, y)
print(f"Hasil: {hasil:.3f}")

File: PRAK204-2510817110006-Muhammad Fadhil Lesmana.c
//Program menghitung: a - b * i / j - x + y
#include <stdio.h>

int main(){
    float a, b, i, j, x, y;
    float hasil;

    printf("Masukkan 6 angka (a b i j x y): ");
    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);

    hasil = a - b * i / j - x + y;

    printf("Hasil: %.3f\n", hasil);
    return 0;
}
